Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+
+**SceneSystem**
+
+MRTK 2.0.0 has added the [Scene System](SceneSystem/SceneSystemGettingStarted.md) to help with
+applications that contain more than once scene.
+
+**Service managers (experimental)**
+
+This release adds service managers to enable the light-weight addition of specific Microsoft 
+Mixed Reality Toolkit features, such as the Spatial Awareness system, individually.
+
+These service managers are imported as part of the Foundation package and are located in the
+MixedRealityToolkit.SDK\Experimental\Features folder and are a work in progress.
+
+Service manager prefabs are provided for the following services.
+
+- BoundarySystem
+- CameraSystem
+- DiagnosticsSystem
+- InputSystem
+- SpatialAwarenessSystem
+- TeleportSystem (requires the Input System)
+
+To use, drag and drop the desired prefab into your heirarchy and select the configuration
+profile. 
+
+> [!Note]
+> These service managers are currently experimental, may have issues and
+are subject to change. Please file any and all issues you encounter on GitHub
+
+**Slate**
+
+- Improved
+    - Usability of slates by adding the shadow based on the finger proximity
+    - ‘Follow Me’ behavior to match the HoloLens 2 shell behavior, using Radial Solver.
+- Fixed
+    - Border thickness issue fixed on flattend Bounding Box
+
+**SpatialObjectMeshObserver**
+
+We have added the SpatialObjectMeshObserver to improve developer productivity when working
+with the Spatial Awareness system. This observer reads mesh data from imported 3D models
+and uses them to simulate environmental data from devices such as Microsoft HoloLens 2.
+
+SpatialObjectMeshObserver is not enabled in the default profiles, please see the
+[Spatial Awareness Gettniving Started](SpatialAwareness/SpatialAwarenessGettingStarted.md) article
+for more information on configuring your application. 
+
+**System Keyboard**
+
+The system keyboard can now be used on all platforms. See the HandInteractionExamples scene in the Examples
+package (Demos\HandInteraction\Scenes\HandInteractionDemos.unity) for a demonstration of using the
+SystemKeyboardExample script.
+
+**Take Screenshot**
+
+A Take Screenshot utility menu item (Mixed Reality Toolkit > Utilities > Take Screenshot) has been added
+to capture high resolution screenshots within the editor. Screenshots can be captured with a transparent
+clear color for use in easy post compositing of images for documentation or media.
+
+**UX Controls**
+
+The following UX controls have been added and/or improved in version 2.0.0. In addition, those in the
+following list can now be instantiated and configured from code.
+
+- BoundingBox
+- ManipulationHandler
+- HandInteractionPanZoom
+- Interactable (basic features)
+
+**Updated Architecture Documentation**
+
+The [archtecture documentation](Architecture/Overview.md) is all new for version 2.0.0.
+
+### Known issues in 2.0.0
+
+The sections below highlight some of the known issues in the Microsoft Mixed Reality Toolkit
```

Comment:
>sections [](start = 4, length = 8)

I think this section deleted the stuff in KnownIssues.md, but didn't copy everything over -there's the MAX_PATH issue, which isn't called out here. Can we do another pass to make sure everything made it over?

Reply:
added. this section had gotten lost in an earlier merge